src/components/js/EcAppCard.js

```diff
@@ -1,8 +1,8 @@
-import { i18n } from '@ecomplus/utils'
+import { _config, i18n } from '@ecomplus/utils'
```

Comment:
```suggestion
import { i18n } from '@ecomplus/utils'
```

Também não vai precisar do `_config` removendo a prop `lang`, certo?

Reply:
ok